src/components/CategoryList.js

```diff
@@ -11,35 +11,34 @@ class CategoryList extends React.Component {
     checkedCategory: 'Housing' // @TODO Add this dynamically.
   };
 
+  componentDidMount() {
+    const { updateActiveCategory } = this.props;
+
+    updateActiveCategory(this.state.checkedCategory);
```

Comment:
To have it consistent, shouldn't this be first initialized like this `const { checkedCategory } = this.state;`

Reply:
you're absolutely right